packages/client/env.ts

```diff
@@ -3,11 +3,13 @@ import { computed } from 'vue'
 import { objectMap } from '@antfu/utils'
 // @ts-expect-error missing types
 import _configs from '/@slidev/configs'
+import { getSlideHeightRespectingAspectAsPossible } from './utils'
 
 export const configs = _configs as SlidevConfig
 export const slideAspect = configs.aspectRatio ?? (16 / 9)
 export const slideWidth = configs.canvasWidth ?? 980
-export const slideHeight = Math.round(slideWidth / slideAspect)
```

Comment:
I think all we need is to change this line. We don't need an extra util on on this.